### ISSUE TITLE
Math Expressions: Equation Numbering and Reference Labels

### ISSUE THREAD

**Issue by roblem** [rhiokim/haroopad#382]
[According to the mathjax website](http://docs.mathjax.org/en/latest/tex.html#automatic-equation-numbering), it should be possible to turn on equation numbering and reference labels but it requires that some parameters be set before calling the mathjax java script.

I tried adding the script supplied at the above site at top of my md file, but didn't work.  Not sure how to implement.
